src/utilities/theme/tokens.ts

```diff
@@ -5,12 +5,6 @@ export const Tokens = {
   borderRadiusWide: rem('8px'),
   borderRadiusFull: '50%',
 
-  // Font-Weights
-  fontWeightRegular: 400,
```

Comment:
Breaking change. Should this removal (along with other tokens here) be adjusted elsewhere?

Reply:
Good callout! I reverted this update 👍  Spoke with @alex and we plan to create a separate issue to remove these tokens and other associated processes.